specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -155,31 +156,54 @@ No direct changes to the public API.
 
 ## Detailed design
 ### VDK Notebook plugin
- This VDK plugin will provide the functionality to run Jobs which will retrieve Job steps from notebook files instead of .py and .sql files. This plugin can be used alone without the JupyterLab extension.
+ This VDK plugin will provide the functionality to run Jobs which contain notebook files instead of .py and .sql files. This plugin can be used alone without the JupyterLab extension.
 As it can be seen from the below diagram the plugin will consist of a new hook and a few new classes.
 
-![vdk-notebook-plugin](vdk-notebook-plugin.png)
+![vdk-notebook-plugin](vdk-notebook-plugin.jpeg)
```

Comment:
How are those pictures produced? I strongly encourage including the "source" so that someone else can edit them if they find a mistake. Or maybe if they want to re-use them and extend them for another VEP for example.

Reply:
Sadly, the tool I am using does not allow such "sharing". But I can put its link, of course. If you have any recommendations for another tool which can provide such sharing of the source would be amazing :)

Reply:
What tool are you using ? What is the link? If I can reproduce it from that link and edit it. This is fine.

Reply:
You can publish it (There's publish button) and include the link as a comment `<!-- link -->` in the VEP next to the image.

Reply:
Yes, but I cannot share everything since there is a limit of 3 documents for the free version. I currently have only one of them as lucid document, others are deleted after work and only the images are saved.

Reply:
I can still link the one I have but I don't think it is ok for long term use since there can be added more images.